### PR TITLE
Dockerfile: symlink scripts

### LINKED ISSUE #50: Symlink for_production.sh and for_production.sh
Inside the container these scripts are no symbolic links to process_images.sh and process_mets.sh but separate scripts. Probably the copy process in the Dockerfile resolves symbolic links. Therefore the linking must be done within the Dockerfile. The existing symbolic links in the repository can be removed.

### PR DESCRIPTION
**Pull request by bertsky**
fixes #50

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -65,6 +65,9 @@ RUN echo AllowUsers ocrd >> /etc/ssh/sshd_config
 RUN echo "cd /data" >> /etc/profile
 RUN /usr/sbin/sshd -t # check the validity of the configuration file and sanity of the keys
 COPY *.sh /usr/bin/
+RUN cd /usr/bin; \
```
Comment (markusweigelt) [inline Dockerfile:68]:
```suggestion
RUN cd /usr/bin \
```

File: Dockerfile [hunk 2]
```diff
@@ -65,6 +65,9 @@ RUN echo AllowUsers ocrd >> /etc/ssh/sshd_config
...
+    ln -fs process_mets.sh for_presentation.sh; \
```
Comment (markusweigelt) [inline Dockerfile:69]:
```suggestion
    ln -fs process_mets.sh for_presentation.sh \
```

**Review by markusweigelt — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (bertsky) [reply Dockerfile:68]: that would not work – you need a cmd separator

**Review by markusweigelt — APPROVED** [review]
Yes you are right! 👍🏻

**Pull request merged**